.gitignore

```diff
@@ -12,3 +12,5 @@ dist/
 # Converted rten ML models
 *.rten
 models/
+
+*.DS_Store
```

Comment:
I think there was a misunderstanding. OS/editor-specific ignores shouldn't be part of this _project_'s `.gitignore`. Rather they belong in your personal global gitignore file. See the blog post I linked to in the earlier comment for how to set that up.

Reply:
I got it. I have updated global gitignore on my machine and restored the .gitignore file in the repository to its original state. Thank you